src/type-coercions.md

```diff
@@ -164,8 +168,7 @@ an implementation of `Unsize<U>` for `T` will be provided:
 
 * `[T; n]` to `[T]`.
 
-* `T` to `U`, when `U` is a trait object type and either `T` implements `U` or
-  `T` is a trait object for a subtrait of `U`.
+* `T` to `dyn U`, when `T` implements `U + Sized`
```

Comment:
There seems to be an unanswered question here: https://github.com/rust-lang/reference/pull/342/files#r191050567

Reply:
Ah. Trait Objects can only be created from Sized types. That said, that's an incomplete restriction, I'll add a line about object safety.